share/completions/magento.fish

```diff
@@ -519,45 +425,20 @@ __fish_magento_register_command_option i18n:pack -s d -l allow-duplicates -d 'Us
 __fish_magento_register_command_option i18n:uninstall -f -s b -l backup-code -d 'Take code and configuration files backup (excluding temporary files)';
 __fish_magento_register_command_option i18n:uninstall -f -a "(__fish_print_magento_languages)" -d 'Language package name';
 
-#
-# indexer:reindex
-#
-__fish_magento_register_command_option indexer:reindex -f -a "(__fish_print_magento_indexer_names)" -d 'Space-separated list of index types or omit to apply to all indexes.';
```

Comment:
You probably still want to complete these commands, even if you can't complete the arguments.